README.md

```diff
@@ -1,65 +1,256 @@
 # MinIO Operator [![Docker Pulls](https://img.shields.io/docker/pulls/minio/k8s-operator.svg?maxAge=604800)](https://hub.docker.com/r/minio/k8s-operator)
 
-MinIO Operator brings native support for [MinIO](https://github.com/minio/minio), [Graphical Console](https://github.com/minio/console), and [Encryption](https://github.com/minio/kes) to Kubernetes. This document explains how to get started with MinIO Operator using `kubectl minio` plugin.
+<a href="https://min.io"> <img src=https://raw.githubusercontent.com/minio/minio/master/.github/logo.svg width="600px"/> </a>
 
-## Prerequisites
+MinIO provides Kubernetes-native high performance object storage for private and
+public cloud infrastructure (Hybrid Cloud). The MinIO API maintains compatibility
+with the S3-API to simplify migration of applications onto MinIO Object Storage.
 
-- Kubernetes >= v1.17.0.
-- Create PVs.
-- Install `kubectl minio` plugin using `kubectl krew install minio`.
+## Table of Contents
 
-## Operator Setup
+* [Architecture](https://github.com/minio/operator#Architecture)
```

Comment:
Hm. These links are broken. Might need to make them explicit as `operator/tree/master/README.md#` and see if that works...